### PR TITLE
Update README.org to be more useful for first-time users

### PR DESCRIPTION
**Pull request by ahendriksen**
The goals was to:
1) Be more clear how to get started with GitHub
2) Explain where to add code (both in examples and main functionality)
3) Explain how to use the examples directory to create one-off scripts
   for personal use.

Do you think this is better?

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -55,20 +53,31 @@ Then:
...
-10. Commit code
-11. Update version in =VERSION= file
-12. Push to github
-13. Create release on GitHub
-14. Create GitHub pages documentation
+10. Write software. You can start by extending the script in
+    =examples/getting_started.py=, or create more scripts in the
+    =examples= directory. Once you identify common functionality, you
+    can add functions to the main module file in
+    =project_slug/project_slug.py=. The =getting_started.py= file
```
Comment (jwbuurlage) [inline README.org]: Is it clear that `project_slug` is replaced by what users put in? We could replace this with e.g. `[your_project_slug]`.

File: README.org [hunk 2]
```diff
@@ -55,20 +53,31 @@ Then:
...
+    contains examples how to call these functions.
+11. Update README
+12. Keep changelog
+13. Update documentation
+    1. make docs
+14. Commit code
+15. Update version in =VERSION= file
+16. Push to github
```
Comment (jwbuurlage) [inline README.org]: GitHub

**Review by jwbuurlage — COMMENTED** [review]
Looks good :+1:

File: README.org (hunk 1 above)
Comment (ahendriksen) [reply README.org]: done

**Pull request merged**